### ISSUE TITLE
Implement message expiry using the Fernet TTL field

### ISSUE THREAD

**Issue by teor2345** [privcount/privcount#85]
If we want to defend against replay attacks, we can do so by encrypting every message using a Fernet symmetric key and use the TTL option to set an expiry time for the message.
(We then encrypt and sign the symmetric key using the RSA signing and encryption keys from #83 and #84.)

The expiry time should be related to the maximum expected delay before the recipient receives the message (including clock skew and network RTT). This could be done at the protocol level in conjunction with #78.

In certain parts of the protocol, a stricter bound might be required, for example, replay attack resistance in #27 requires that all messages from previous rounds expire before starting a new round. This can be achieved using shorter expiry times, or longer gaps between rounds. This could be done at the message level in conjunction with #27.

**Comment by teor2345** [comment]
This would be WONTFIX on its own, but it might end up being a convenient way of implementing #27.
